cypress/e2e/smoke/assets.cy.js

```diff
@@ -0,0 +1,222 @@
+  })
+
+  it('Verify that non-native tokens are present and have balance', () => {
+    balances.selectTokenList(balances.tokenListOptions.allTokens)
+    balances.verifyBalance(balances.currencyDaiCap, TOKEN_AMOUNT_COLUMN, balances.currencyDaiAlttext)
+    balances.verifyTokenBalanceFormat(
+      balances.currencyDaiCap,
+      balances.currencyDaiFormat_2,
+      TOKEN_AMOUNT_COLUMN,
+      FIAT_AMOUNT_COLUMN,
+      fiatRegex,
+    )
+
+    balances.verifyBalance(balances.currencyAave, TOKEN_AMOUNT_COLUMN, balances.currencyAaveAlttext)
+    balances.verifyTokenBalanceFormat(
+      balances.currencyAave,
+      balances.currentcyAaveFormat,
+      TOKEN_AMOUNT_COLUMN,
+      FIAT_AMOUNT_COLUMN,
+      fiatRegex,
+    )
+
+    balances.verifyBalance(balances.currencyLink, TOKEN_AMOUNT_COLUMN, balances.currencyLinkAlttext)
+    balances.verifyTokenBalanceFormat(
+      balances.currencyLink,
+      balances.currentcyLinkFormat,
+      TOKEN_AMOUNT_COLUMN,
+      FIAT_AMOUNT_COLUMN,
+      fiatRegex,
+    )
+
+    balances.verifyBalance(balances.currencyTestTokenA, TOKEN_AMOUNT_COLUMN, balances.currencyTestTokenAAlttext)
+    balances.verifyTokenBalanceFormat(
+      balances.currencyTestTokenA,
+      balances.currentcyTestTokenAFormat,
+      TOKEN_AMOUNT_COLUMN,
+      FIAT_AMOUNT_COLUMN,
+      fiatRegex,
+    )
+
+    balances.verifyBalance(balances.currencyTestTokenB, TOKEN_AMOUNT_COLUMN, balances.currencyTestTokenBAlttext)
+    balances.verifyTokenBalanceFormat(
+      balances.currencyTestTokenB,
+      balances.currentcyTestTokenBFormat,
+      TOKEN_AMOUNT_COLUMN,
+      FIAT_AMOUNT_COLUMN,
+      fiatRegex,
+    )
+
+    balances.verifyBalance(balances.currencyUSDC, TOKEN_AMOUNT_COLUMN, balances.currencyTestUSDCAlttext)
+    balances.verifyTokenBalanceFormat(
+      balances.currencyUSDC,
+      balances.currentcyTestUSDCFormat,
+      TOKEN_AMOUNT_COLUMN,
+      FIAT_AMOUNT_COLUMN,
+      fiatRegex,
+    )
+  })
+
+  it('Verify that every token except the native token has a "go to blockexplorer link"', () => {
+    balances.selectTokenList(balances.tokenListOptions.allTokens)
+    // Specifying true for Sepolia. Will delete the flag once completely migrate to Sepolia
+    balances.verifyAssetNameHasExplorerLink(balances.currencyUSDC, ASSET_NAME_COLUMN, true)
+    balances.verifyAssetNameHasExplorerLink(balances.currencyTestTokenB, ASSET_NAME_COLUMN, true)
+    balances.verifyAssetNameHasExplorerLink(balances.currencyTestTokenA, ASSET_NAME_COLUMN, true)
+    balances.verifyAssetNameHasExplorerLink(balances.currencyLink, ASSET_NAME_COLUMN, true)
+    balances.verifyAssetNameHasExplorerLink(balances.currencyAave, ASSET_NAME_COLUMN, true)
+    balances.verifyAssetNameHasExplorerLink(balances.currencyDaiCap, ASSET_NAME_COLUMN, true)
+    balances.verifyAssetExplorerLinkNotAvailable(constants.tokenNames.sepoliaEther, ASSET_NAME_COLUMN)
+  })
+
+  it('Verify that the default currency and the changing effects', () => {
```

Comment:
I misspelled this one in TestRail myselfe. Replace it for "Verify the default Fiat currency and the effects after changing it"